src/Core/src/Platform/Android/MauiScrollView.cs

```diff
@@ -187,9 +187,14 @@ protected override void OnLayout(bool changed, int left, int top, int right, int
 		{
 			base.OnLayout(changed, left, top, right, bottom);
 
-			if (_hScrollView != null && _hScrollView.Parent == this)
+			if (_hScrollView != null && _hScrollView.Parent == this && _content != null)
```

Comment:
```suggestion
			if (_hScrollView?.Parent == this && _content is not null)
```
We can simplify the if, right? null is not this